Dockerfile

```diff
@@ -20,4 +20,5 @@ RUN install-requirements.py -d ~/docker-base/base-requirements.txt -r /tmp/insta
 # Install the current package
 COPY . /tmp/install/katsdpdata
 WORKDIR /tmp/install/katsdpdata
-RUN python ./setup.py clean && pip install --no-index .
+RUN python ./setup.py clean
+RUN pip install --no-deps .
```

Comment:
This is the approach I've been using, but I also add [`pip check`](https://github.com/ska-sa/katsdpingest/blob/8ac4fc1023189b83c58c8dfa6a23731a424d51d6/Dockerfile#L15) afterwards to validate that there weren't any dependencies I forgot about

Reply:
@bmerry thanks. Good plan.  I'll do that too.